cockatrice/src/tab_room.cpp

```diff
@@ -185,12 +185,17 @@ QString TabRoom::sanitizeHtml(QString dirty) const
 
 void TabRoom::sendMessage()
 {
-    if (sayEdit->text().isEmpty()){
+    if (sayEdit->text().isEmpty())
+    {
```

Comment:
I'm guessing your clion style config is wrong; make sure it's set to open braces same line as 'if'.

settings > editor > code style > c++

<img width="352" alt="screen shot 2017-03-18 at 7 50 19 am" src="https://cloud.githubusercontent.com/assets/14644/24071725/9623890a-0baf-11e7-8e2e-ed3de868c39b.png">

Specifically 'other' controls this